### DISCUSSION TITLE
Style guide: Better markup for UI elements?

### DISCUSSION THREAD

**Discussion by hamishwillee** [mdn/mdn-community#591] [Content]
The MDN guidelines for talking about UI actions is ([from here](https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Writing_style_guide#user_interface_actions)):

----

In task sequences, describe user interface actions using the imperative mood. Identify the user interface element by its label and type.

- **Correct:** Click the Edit button.
- **Incorrect:** Click Edit.

----

This guidance is good, but my experience is that UI elements are one of the cases where readers almost always benefit from using styling on the UI element name.

I've just been looking at a PR with this text:

> In Firefox: Open the [about:debugging](/en-US/docs/Tools/about:debugging) page, click the This Firefox option, click the Load Temporary Add-on button, then select any file in your extension's directory.

I much prefer bold for the action as this makes it easier to see.

> In Firefox: Open the [about:debugging](/en-US/docs/Tools/about:debugging) page, click the **This Firefox option**, click the **Load Temporary Add-on** button, then select any file in your extension's directory.

Markup should always be used sparingly, but this is a case where it can make things easier to parse. Even the simple case is better:
- Click the **Edit** button.

Anyone else think this is better/worth changing the guidelines for?

**Comment by pepelsbey** [comment]
Closing this discussion as it didn’t get much traction. Let me know if you’d like to revive it after all.
